src/Containers/EditControlPanel/EditControlPanel.stories.tsx

```diff
@@ -23,6 +24,14 @@ const handleCloneClick = () => {
     console.log('Clone Button has been clicked');
 };
 
+const handleRemoveChairClick = () => {
+    console.log('- Button has been clicked');
+};
+
+const handleAddChairClick = () => {
+    console.log('+ Button has been clicked');
+};
+
 const Template: Story<IEditControlPanel> = (args) => (
```

Comment:
why not just use Args Types so it shows up in the STorybook Actions (Makes it eaiser to debug)

Reply:
Good idea, I will convert to use Arg Types after I finish the user story to edit the amount of chairs for a table